TesseractCore/Importing.swift

```diff
@@ -0,0 +1,57 @@
+//  Copyright (c) 2015 Rob Rix. All rights reserved.
+
+private func unique<T: Hashable>(list: [T]) -> [T] {
+    return Array(Set(list))
+}
+
+public func parseEdge(edge: String) -> (String, String)? {
+    let characterParser = %("0"..."9") | %("a"..."z") | %("A"..."Z")
+    let ignoreBeginningOfLine = ignore("\t") ++ ignore("\"")
+    let edgeParser = ignoreBeginningOfLine ++ characterParser+ ++ ignore("\" -> \"") ++ characterParser+ ++ ignore("\";");
+    if let (rawSource, rawDestination) = edgeParser(edge)?.0 {
+        let source = String(rawSource.reduce([], combine: +))
+        let destination = String(rawDestination.reduce([], combine: +))
+        return (source, destination)
+    }
+    
+    return nil
+}
+
+public func importDOT(file: String) -> Graph<String> {
+    let lines = split(file, { $0 == "\n" })
+    let rawLines = lines[1...(lines.count - 2)]
```

Comment:
It’d be good to document why we want to skip the first line. Actually, a comment with a link to the spec would be pretty much ideal!